portal-web/docroot/html/portlet/layouts_admin/edit_layout_set.jsp

```diff
@@ -69,43 +61,9 @@ boolean hasViewPagesPermission = (pagesCount > 0) && (liveGroup.isStaged() || se
 
 			<aui:nav-item href="<%= addPagesURL %>" iconCssClass="icon-plus" label="add-page" />
 		</c:if>
-		<c:if test="<%= hasExportImportLayoutsPermission %>">
-			<liferay-portlet:renderURL portletName="<%= PortletKeys.EXPORT_IMPORT %>" var="exportPagesURL" windowState="<%= LiferayWindowState.POP_UP.toString() %>">
-				<portlet:param name="struts_action" value="/export_import/export_layouts" />
-				<portlet:param name="<%= Constants.CMD %>" value="<%= Constants.EXPORT %>" />
-				<portlet:param name="groupId" value="<%= String.valueOf(groupId) %>" />
-				<portlet:param name="liveGroupId" value="<%= String.valueOf(liveGroupId) %>" />
-				<portlet:param name="privateLayout" value="<%= String.valueOf(privateLayout) %>" />
-				<portlet:param name="rootNodeName" value="<%= rootNodeName %>" />
-				<portlet:param name="showHeader" value="<%= Boolean.FALSE.toString() %>" />
-			</liferay-portlet:renderURL>
-
-			<aui:nav-item href="<%= exportPagesURL %>" iconCssClass="icon-arrow-down" label="export" useDialog="<%= true %>" />
-
-			<liferay-portlet:renderURL portletName="<%= PortletKeys.EXPORT_IMPORT %>" var="importPagesURL" windowState="<%= LiferayWindowState.POP_UP.toString() %>">
-				<portlet:param name="struts_action" value="/export_import/import_layouts" />
-				<portlet:param name="<%= Constants.CMD %>" value="<%= Constants.VALIDATE %>" />
-				<portlet:param name="groupId" value="<%= String.valueOf(groupId) %>" />
-				<portlet:param name="privateLayout" value="<%= String.valueOf(privateLayout) %>" />
-				<portlet:param name="rootNodeName" value="<%= rootNodeName %>" />
-				<portlet:param name="showHeader" value="<%= Boolean.FALSE.toString() %>" />
-			</liferay-portlet:renderURL>
-
-			<aui:nav-item href="<%= importPagesURL %>" iconCssClass="icon-arrow-up" label="import" useDialog="<%= true %>" />
-		</c:if>
 	</aui:nav>
 </aui:nav-bar>
 
-<c:if test="<%= liveGroup.isStaged() %>">
-	<%@ include file="/html/portlet/export_import/error_auth_exception.jspf" %>
-
-	<%@ include file="/html/portlet/export_import/error_remote_export_exception.jspf" %>
-
-	<div class="alert alert-warning">
```

Comment:
It was removed intentionally, because this message should be generic for all the portlets on control panel, not only for layouts admin.

Could you please take care of this? If you need any help please let me know.